tests/models/dstability/test_dstability_model.py

```diff
@@ -115,6 +115,8 @@ def test_given_data_when_parseandserialize_then_doesnotraise(self, dir_path: str
         [
             pytest.param("dstability/example_1", id="Input Structure"),
             pytest.param("dstability/Tutorial_v20_2_1", id="Tutorial DStability 20.2.1"),
+            pytest.param("dstability/Tutorial_v2022_1_1", id="Tutorial DStability 2022.1"),
```

Comment:
Small inconsistency with the version; based on the changes I would expect: `2022.01`.